CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.7.1](https://github.com/nf-core/sarek/releases/tag/2.7.1) - Pårtejekna
+
+Pårtejekna is one of glaciers of the Pårte Massif.
+
+### Added
+
+- [#353](https://github.com/nf-core/sarek/pull/353) - Add support for task retries with exit code 247 (exhibited by Picard MarkDuplicates)
+
```

Comment:
Looks like MSISensor-Pro addition, the controlfreec changes, and Mutect2 Single is missing here.

Reply:
Good point...
I'll fix that in a separate PR

Reply:
Yes, I don't see that stuff in the PR

Reply:
CHANGELOG is now up to date